backend/services/schema_service.py

```
@@ -0,0 +1,341 @@
+            output_path = await self._save_schema_files(connection, cards)
+            
+            conn.close()
+            
+            return {
+                "success": True,
+                "message": f"{len(cards)}개 테이블 스키마 추출 완료",
+                "output_path": str(output_path),
+                "table_count": len(cards)
+            }
+            
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)